spotlight-client/src/SiteNavigation/SiteNavigation.test.tsx

```diff
@@ -0,0 +1,112 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import { fireEvent, screen, within } from "@testing-library/react";
+import useBreakpoint from "@w11r/use-breakpoint";
+import { runInAction } from "mobx";
+import React from "react";
+import DataStore from "../DataStore";
+import { renderWithStore } from "../testUtils";
+import SiteNavigationContainer from ".";
+
+jest.mock("@w11r/use-breakpoint");
+
+const useBreakpointMock = useBreakpoint as jest.Mock;
+
+beforeEach(() => {
+  useBreakpointMock.mockReturnValue(true);
+  runInAction(() => {
+    DataStore.tenantStore.currentTenantId = "US_ND";
```

Comment:
I did notice that the behavior here is confusing when there isn't a tenant selected and the menu button is clicked.. I'm sure support for that is coming, just wanted to flag it.

Reply:
yeah at the moment if you do that the menu won't have anything in it ... that will change eventually